workflows/crisprseq_screening.nf

```diff
@@ -125,19 +139,57 @@ workflow CRISPRSEQ_SCREENING {
         )
         ch_versions = ch_versions.mix(FASTQC.out.versions.first())
 
-        ch_input_cutadapt = ch_input.combine(Channel.value([[]]))
+        //set adapter seq to null to make it compatible with crispr targeted
+        ch_cutadapt = ch_input.combine(Channel.value([[]]))
 
-    if(params.cutadapt) {
-        CUTADAPT(
-            ch_input_cutadapt
-        )
-        ch_versions = ch_versions.mix(CUTADAPT.out.versions)
+        if(params.five_prime_adapter) {
+            CUTADAPT_FIVE_PRIME(
+                ch_cutadapt
+            )
+            CUTADAPT_FIVE_PRIME.out.reads.combine(Channel.value([[]])).set { ch_cutadapt }
```

Comment:
I would prefer to not override the channels and use a different name, this could cause problems.
The channel is called the same for cutadapt input, and output. And it is again called the same if we run cutadapt_five_prime and cutadapt_three_prime